cmd/setup/52.sql

```diff
@@ -0,0 +1,14 @@
+DO
+$$
+    BEGIN
+        IF NOT EXISTS
+            (SELECT 1
+             FROM information_schema.tables
+             WHERE table_schema = 'projections'
+               AND table_name = 'idp_templates6_ldap2')
+        THEN
+            ALTER TABLE IF EXISTS projections.idp_templates6_ldap3 RENAME COLUMN rootCA TO root_ca;
+            ALTER TABLE IF EXISTS projections.idp_templates6_ldap3 RENAME TO idp_templates6_ldap2;
+        END IF;
+    END
+$$;
```

Comment:
exists check is done by the alter already
```suggestion
ALTER TABLE IF EXISTS projections.idp_templates6_ldap3 RENAME COLUMN rootCA TO root_ca;
ALTER TABLE IF EXISTS projections.idp_templates6_ldap3 RENAME TO idp_templates6_ldap2;
```

Reply:
It is not, as I only want to try to rename `idp_templates6_ldap3` to `idp_templates6_ldap2` with the `ALTER TABLE`, if there is no `idp_templates6_ldap2` already existing.

Reply:
ah got it

Reply:
but there will never be a ldap2 table if ldap3 exists so the proposed changes should still be valid